al_hostserver/altrain.py

```diff
@@ -284,6 +287,9 @@ def parse_args(argv):
                         help="Specifies the directory of the outer loop repo.")
     parser.add_argument('--outer-loop-url', default=None, dest="outer_loop_url",      metavar="<outer_loop_url>",
                         help="Specifies the URL of a running outer loop server.")
+    parser.add_argument('--debug', action='store_true', help="Makes logging messages visible in the console.")
```

Comment:
How is the behavior of this flag different form other debugging flags? It seems like this one corresponds to debugging in the host_server specifically. Can it be given a more descriptive name? Alternatively, see my other comment that it could maybe be part of a delimited list of loggers to activate.